src/Command.js

```diff
@@ -166,7 +174,10 @@ export default class Command {
   }
 
   runValidations() {
-    if (!GitUtilities.isInitialized(this.repository.rootPath)) {
+    const opts = Object.assign({}, this.execOpts, {
+      cwd: this.repository.rootPath
```

Comment:
I'm not sure I get what you are referring to :/ `GitUtilities` only has static methods, how would it have execopts preloaded into it?